### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by AntonMurashov**

### CODE REVIEW

File: src/app/courses/add-course/add-course.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="col caption">Duration</div>
+    <div class="row">
+        <input pattern="[0-9]*" class="input date" [(ngModel)]="duration">
+        <div class="duration">{{duration | duration}}</div>
+    </div>
+
+    <div class="col caption">Authors</div>
+    <input class="row input" [(ngModel)]="authors">
+
+    <button class="button save-button" [disabled]="isSaveDisabled()" (click)="onSaveClick()" [ngClass]="{'button-disabled':isSaveDisabled() === true}">Save</button>
```
Comment (AinuraAliieva) [inline src/app/courses/add-course/add-course.component.html]: jfi. also we could use 
```
[class.button-disabled]="isSaveDisabled()"
```

File: src/app/courses/add-course/add-course.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      topRated: false
+    });
+    this.onClosePage.emit(this.courseService.findAll());
+  }
+
+  public onCancelClick() {
+    this.onClosePage.emit();
+  }
+
+  private parseDate(date: string): Date {
```
Comment (AinuraAliieva) [inline src/app/courses/add-course/add-course.component.ts]: should we use `parseDate` from `DateService ` ?

**Comment by AinuraAliieva** [comment]
Please enable AOT and fix issues in the templates that revealed after enabling AOT

File: src/app/courses/add-course/add-course.component.ts (hunk 2 above)
Comment (AntonMurashov) [reply src/app/courses/add-course/add-course.component.ts]: Created a service especially for this, but forgot to use ))  Fixed

File: src/app/courses/add-course/add-course.component.html (hunk 1 above)
Comment (AntonMurashov) [reply src/app/courses/add-course/add-course.component.html]: Your code looks much better! Changed!

**Pull request merged**